src/Appium.Net/Appium/ImageComparison/OccurenceMatchingOptions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Collections.Generic;
+
+namespace OpenQA.Selenium.Appium.ImageComparison
+{
+    public class OccurenceMatchingOptions : ComparisonOptions
+    {
+        public override Dictionary<string, object> GetParameters()
```

Comment:
There is also threshold option here, which has been added to java client later
https://github.com/appium/java-client/blob/e19ac644699daf8cc6746f37a1bd86fb3c705e16/src/main/java/io/appium/java_client/imagecomparison/OccurrenceMatchingOptions.java#L34